frontend/src/pages/UI/Breadcrumb.jsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+
+const BreadcrumbList = (props) => {
```

Comment:
Mind if we split `props`?  I.e. something like this...
```suggestion
const BreadcrumbList = ({isCurrent, children}) => {
```

Reply:
Using `props` I think is the idiomatic way of passing it in: https://reactjs.org/docs/components-and-props.html
Our existing code uses `props` this way as well.

Reply:
Mmmm.  I'm going to push back just a bit more.  If you really think `props` is the way to go, I'll approve.  But...  I see a number blob posts mention destructuring the arguments is a good practice, we did this to great effect at my previous client (US Tax Court), and I think this helps with us understanding specifically what is needed for a component to operate with a quick glance.

Reply:
I found a couple examples of it being done this way online as well - I'll apply it since I don't have a strong preference.